articles/ai-services/computer-vision/whats-new.md

```diff
@@ -17,6 +17,15 @@ ms.author: pafarley
 
 Learn what's new in the service. These items may be release notes, videos, blog posts, and other types of information. Bookmark this page to stay up to date with new features, enhancements, fixes, and documentation updates.
 
+## September 2023
+
+### Retirement of outdated Computer Vision API versions
+
+Computer Vision API versions 1.0, 2.0, 3.0, and 3.1 will be deprecated on September 13, 2026. Developers won’t be able to make API calls to these APIs after that date.
```

Comment:
I think you have the terms switched: "deprecation" is now, and they'll be retired in 2026. Right?